packages/blade/package.json

```diff
@@ -135,6 +136,7 @@
     "@babel/preset-react": "7.16.5",
     "@babel/preset-typescript": "7.16.5",
     "@codesandbox/sandpack-react": "1.16.0",
+    "@emotion/react": "11.11.1",
```

Comment:
Oh wait, does that mean now consumers will have bundle of both styled comp and emotion?

Reply:
Yup thats a down side of this library. Mentioned it in the API decision 😢 

Doesnt add much though

Reply:
I see, so does the library marks @emotion/react as peerDep? 

It should otherwise all the components will have to carry emotion bundle this is what I was exploring also. 

Also, let's move `@table-library/react-table-library` to peerDep, we don't need to include this on our bundle if a consumer doesn't use Table keeping in mind treeshaking/codesplitting things.

Reply:
Checking